packages/graphback-cli/src/helpers/db.ts

```diff
@@ -16,12 +19,18 @@ const handleError = (err: { code: string; message: string; }): void => {
 
 export const dropDBResources = async(): Promise<void> => {
   try {
-    const { dbConfig } = JSON.parse(readFileSync(`${process.cwd()}/config.json`, 'utf8'))
-    const manager = new PostgresSchemaManager(dbConfig);
-    // tslint:disable-next-line: await-promise
-    await manager.getConnection().raw('DROP SCHEMA public CASCADE;')
-    // tslint:disable-next-line: await-promise
-    await manager.getConnection().raw('CREATE SCHEMA public;')
+    const { database, dbConfig } = JSON.parse(readFileSync(configPath, 'utf8'))
+    if(database === 'sqlite3') {
+      if(existsSync(`${process.cwd()}/db.sqlite`)) {
```

Comment:
I think we need configuration where this DB will be placed relative to project.
Not in this PR but great to have separate issue